decentraland/unity-explorer
# feat: emote wheel

## What does this PR change?

Implements the emote wheel.

<img width="1726" alt="Screenshot 2024-04-24 at 16 21 03" src="https://github.com/decentraland/unity-explorer/assets/56365551/5cd2c074-e8f6-45b0-a883-156522021367">

## How to test the changes?

1. Launch the explorer
2. Press `B`
3. The emote wheel should be opened
4. Check that the slots configuration are OK compared to the backpack configuration
5. Press any of the numbers `[0~9]` and observe that the emote should be played
6. Click on any of the slots and observe that the emote should be played
7. Press `E` or click the middle of the wheel. The backpack should be opened on the emotes section
8. Close the wheel
9. Press `B+num`. Observe that the emote is played but the emote wheel is not opened
10. Change emotes in the backpack and reopen the wheel. Check that the slots configuration is OK
11. Check that the emote rarity backgrounds match with its rarity

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
